modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,15 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
 
-If you cannot use floating IP addresses, the {product-title} installation might still finish. However, the installation program fails after it times out waiting for API access.
+//TODO Odd to write about, as it's odd UX. File a bug.
```

Comment:
It might be good to mention that if the instance that you are running the installer cannot reach the api, then the installer will time out. This is what causes the case described below.

Reply:
there are ways around this like proxying and being on the same network as the cluster

Reply:
> It might be good to mention that if the instance that you are running the installer cannot reach the api, then the installer will time out. This is what causes the case described below.

@iamemilio Does line 13 accomplish this?

> there are ways around this like proxying and being on the same network as the cluster

So, essentially:

You can mitigate the need for FIPs w/proxies or ensuring that you use the same network as the cluster. Otherwise, the installer will time out, and you'll need to perform post-installation network configuration. 

?

Reply:
Yep!

Reply:
LGTM then